.gitpod.yml

```diff
@@ -6,6 +6,7 @@ tasks:
   - name: Terminal
     openMode: split-left
     init: |
+      eval $(gp env -e gp env RWFW_PATH="/workspace/redwood")
```

Comment:
@dac09 I'm having some trouble getting this working; `RWFW_PATH` seems to be undefined for me. Was there a reason for `gp env` again after `gp env -e` here?

Reply:
Updated - sorry typo :)